openquake/risklib/qa_tests/event_based_test.py

```diff
@@ -88,7 +88,7 @@ def test_mean_based_with_partial_correlation(self):
         # This is a regression test. Data has not been checked
         vf = (
             scientific.VulnerabilityFunction(
-                'PGA',
+                'SOME-TAXONOMY', 'PGA',
```

Comment:
No, because the ID is ignored in such tests, however, let me fix this, to avoid any possible confusion.